aboutme.html

```diff
@@ -0,0 +1,48 @@
+<!Doctype html>
+<html lang="en" ontouchmove>
+	<head>
+		<title>Portfolio ALex F.</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link href="./css/normalise.css" type="text/css" rel="stylesheet">
+		<link href="./css/style.css" type="text/css" rel="stylesheet">
+		
+	</head>
+	
+	<header>
+		<nav class="container navbar">
+			<h1>Alex Ferreira</h1>
+			<div class="link">
+				<a href="./index.html" class="navlink">Home</a>
+				<a href="./portfolio.html"class="navlink ">Portfolio</a>
+				<a href="./aboutme.html"class="navlink current">About me</a>
+				<a href="./contact.html"class="navlink">Contact</a>
+			</div>
+			
+		</nav>
+	</header>
+	
+	<body class="body-image">
+		<main id="aboutme-body-image">
+			
+			<div class="astronaute">
+				<img src="./images-logo/astronaute.png" alt="image of an Astronaute">
+			</div>
+			
+			<div class="content-box aboutme">
+				<h2>-About Me-</h2>
+				<div class="aboutme-div">
```

Comment:
`aboutme-div` isnt a very semantic class name. what if you want to put it on something that isn't a div?